PKGBUILD

```diff
@@ -38,5 +44,8 @@ package() {
 
   cd "${srcdir}/${pkgname}-${pkgver}"
   python setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
+  
   install -D -m644 COPYING "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+  install -D -m644 "${srcdir}/${pkgname}.desktop" "${pkgdir}/usr/share/applications/${pkgname}.desktop"
+  install -D -m644 "${srcdir}/${pkgname}-${pkgver}/data/${pkgname}_halo.svg" "${pkgdir}/usr/share/pixmaps/${pkgname}.svg"
```

Comment:
Well, I have 130 icons in pixmaps and 200 in 32x32/apps, so it seems balanced. We should follow the Icon Theme Specification imho.